includes/Modules/Analytics/Advanced_Tracking.php

```diff
@@ -79,14 +97,25 @@ public function register() {
 			function() {
 				$this->set_up_advanced_tracking();
 			},
-			11
+			15
 		);
 		add_filter(
 			'googlesitekit_amp_gtag_opt',
 			function( $gtag_amp_opt ) {
 				return $this->set_up_advanced_tracking_amp( $gtag_amp_opt );
 			}
```

Comment:
As far as I can tell from the logic here, tracking these events in AMP will currently not work. That is because the logic to register the event lists only runs for non-AMP right now.

However, the problem here is that the logic to print the AMP version (see Site Kit's `Analytics::print_amp_gtag` method) only runs on `wp_footer`, which is too late to register all the event list logic. We will probably need to modify a part of the Site Kit `Analytics` class to make this possible. Let's add a TODO comment here for now and discuss this next week.

Reply:
The TODO comment has been added.